test-apps/imodel-transformer/src/Main.ts

```diff
@@ -49,9 +50,16 @@ interface CommandLineArgs {
   excludeCategories?: string;
 }
 
+const usage = `
+Transform the specified source iModel into a new target iModel.
+You must set up a .env file to connect to an online iModel, see the .env.template file to do so.
+`.trim();
+
 void (async () => {
   try {
-    Yargs.usage("Transform the specified source iModel into a new target iModel");
+    dotenv.config({ path: path.resolve(__dirname, "../.env")});
```

Comment:
you may want to use dotenv-expand as well since that's what we use in most other places. Please may have their .env already setup with that syntax.

Reply:
fair point, I'll add that

Reply:
done, please check and resolve if correct